app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt

```diff
@@ -290,6 +291,17 @@ class PatientsTabScreen : BaseScreen<
     router.push(PatientSummaryScreenKey(patientId, OpenIntention.ViewExistingPatient, Instant.now(utcClock)))
   }
 
+  override fun openSimpleOnPlaystore() {
+    val packageManager = requireContext().packageManager
+    val intent = Intent(Intent.ACTION_VIEW, Uri.parse(PLAY_STORE_URL_FOR_SIMPLE))
+
+    if (intent.resolveActivity(packageManager) != null) {
+      requireContext().startActivity(intent)
+    } else {
+      CrashReporter.report(ActivityNotFoundException("Unable to play store url because no supporting apps were found."))
```

Comment:
It looks like we missed "open" from the error message. Can you add that, please?

```
Unable to open the play store URL...
```